app/src/main/res/values/strings.xml

```diff
@@ -14,6 +14,7 @@
     <string name="title_settings">Settings</string>
     <string name="title_password">Change Password</string>
     <string name="title_update_profile">Update profile</string>
+    <string name="title_alert_dialog">Alert</string>
```

Comment:
What about adding these strings to our Hindi file too?

Reply:
Cool, I'll add them :+1: